shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4

```diff
@@ -1903,3 +1903,13 @@ scnTimestampClause
 restorePointClause
     : RESTORE POINT restorePoint
     ;
+
```

Comment:
The scnTimestampExpr should be considered. 

```
TO SCN Clause

Specify the system change number (SCN) corresponding to the point in time to which you want to return the table. The expr must evaluate to a number representing a valid SCN.

TO TIMESTAMP Clause

Specify a timestamp value corresponding to the point in time to which you want to return the table. The expr must evaluate to a valid timestamp in the past. The table will be flashed back to a time within approximately 3 seconds of the specified timestamp.
```
Whether scnTimestampExpr could satisfy the sentence below.
```
FLASHBACK TABLE employees_test TO TIMESTAMP (SYSTIMESTAMP - INTERVAL '1' minute);

```

Reply:
yes, `FLASHBACK TABLE employees_test TO TIMESTAMP (SYSTIMESTAMP - INTERVAL '1' minute);` is supported

![image](https://user-images.githubusercontent.com/66914151/128764248-708ff294-1929-423f-bf00-a4eb840fae79.png)

`timestampValue` is defined as following:

```
scnValue
    : literals
    ;

timestampValue
    : LP_? expr+ RP_?
    ;

scnTimestampExpr
    : scnValue | timestampValue
    ;
```
´